packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -416,7 +416,7 @@ export class FilesService {
             },
         );
 
-        const result: { rows: Array<{ row_number: string }> } = await this.filesRepository.createQueryBuilder().raw(
+        const result: { rows: Array<{ row_number: string }> } = await this.filesRepository.getKnex().raw(
```

Comment:
Why this change to knex? I couldn't find anything regarding this in the migration guide

Reply:
The raw helper was removed from the query builder:

> The previous em.raw() and qb.raw() helpers are now removed in favor of this new static raw helper. Similarly, the expr helper is also removed in favor of it.
> – https://mikro-orm.io/docs/upgrading-v5-to-v6#raw-sql-fragments-now-require-raw-helper